### ISSUE TITLE
popover.js:32 Uncaught TypeError: Cannot read property 'Default' of undefined

### ISSUE THREAD

**Issue by skwalgit** [twbs/bootstrap#19937]
I'm trying bootstrap 4 with lumen using my own gulp but when I tried a sample page and inspected the page. I got a JS error. Not sure if this is a bug or if I made some mistake. I tried to remove popover.js and tooltip.js and I've got no errors at all.

**Comment by cvrebert** [comment]
(Duplicate of #17574)
@skwalgit Please post a JS Bin, JS Fiddle, or similar that demonstrates the problem.

**Comment by skwalgit** [comment]
@cvrebert Nope, my problem is not a duplicate of that issue. I used the correct destination path and also tried to use the js/dist files but when I tried the js/dist files it returned gulp errors. I am using the js/src folder. I also tried to change the order of the js scripts in my manifest override hoping that popover.js has a dependency and my order was wrong but the error still occurs.

**Comment by cvrebert** [comment]
@skwalgit Anyway, in order to help you, we'll need an example or step-by-step instructions that demonstrate the problem.

**Comment by skwalgit** [comment]
bower install --save bootstrap#v4.0.0-alpha.2
bower.json overrides looks like this
"overrides": {
    "bootstrap": {
      "main": [
        "./scss/bootstrap-flex.scss",
        "./js/dist/alert.js",
        "./js/dist/button.js",
        "./js/dist/carousel.js",
        "./js/dist/collapse.js",
        "./js/dist/dropdown.js",
        "./js/dist/modal.js",
        "./js/dist/popover.js",
        "./js/dist/scrollspy.js",
        "./js/dist/tab.js",
        "./js/dist/tooltip.js",
        "./js/dist/util.js"
      ]
    }
  }

Everything is working fine if i remove popover.js and tooltip.js

Thanks!

**Comment by cvrebert** [comment]
We can't really help with gulp stuff... Can this be reproduced without gulp?

**Comment by skwalgit** [comment]
I've got it. tooltip.js needs tether.js. Error is gone when I included a tether.js

**Comment by cvrebert** [comment]
Glad to hear you found a solution.

**Comment by sGerli** [comment]
I am having this same bug with the lastest alpha, Chrome says: 
`Uncaught TypeError: Cannot read property 'Default' of undefined`
And I included JQuery and Tether before Bootstrap
**Note** I have concatenated all Bootstrap 4 JS using the following grunt:
```
concat: {
        options: {
            separator: ';',
            stripBanners: true,
             banner: '/*! <%= pkg.name %> <%= grunt.template.today("yyyy-mm-dd") %> */\n'
        },
 
        dist: {
            src: ['node_modules/bootstrap/js/dist/**/*.js'],
            dest: 'site/js/bootstrap.min.js'
        }
    },
```

**Comment by sGerli** [comment]
Here's a js fiddle where the problem is shown: [https://jsfiddle.net/yfazs31z/2/](https://jsfiddle.net/yfazs31z/2/)

**Comment by twbs-lmvtfy** [comment]
Hi @sGerli!

You appear to have posted a live example (https://fiddle.jshell.net/yfazs31z/2/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 3773, column 11 thru column 125: Element ```` img ```` is missing required attribute ```` src ````.
* line 3785, column 11 thru column 125: Element ```` img ```` is missing required attribute ```` src ````.
* line 3797, column 11 thru column 125: Element ```` img ```` is missing required attribute ```` src ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by sGerli** [comment]
@twbs-lmvtfy the HTML code from the example is a copy and paste from the official [Bootstrap 4 Examples ](https://v4-alpha.getbootstrap.com/examples/).

**Comment by sGerli** [comment]
Updated link is the same: [https://jsfiddle.net/yfazs31z/2/](https://jsfiddle.net/yfazs31z/2/)

**Comment by twbs-lmvtfy** [comment]
Hi @sGerli!

You appear to have posted a live example (https://fiddle.jshell.net/yfazs31z/2/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 3773, column 11 thru column 125: Element ```` img ```` is missing required attribute ```` src ````.
* line 3785, column 11 thru column 125: Element ```` img ```` is missing required attribute ```` src ````.
* line 3797, column 11 thru column 125: Element ```` img ```` is missing required attribute ```` src ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)
